static/js/containers/ChannelPage.js

```diff
@@ -111,9 +113,11 @@ export class ChannelPage extends React.Component<ChannelPageProps> {
       subscribedChannels,
       posts,
       location: { search },
-      renderPosts
+      renderPosts,
+      isModerator
     } = this.props
 
+    const isAbout = match.path === "/c/:channelName/about/"
```

Comment:
This could be done more simply with nested `<Route/>` tags, but I don't think you can do that here without a non-trivial refactor of how the other pages under /c/:channelName/ are routed too. I'll open a ticket for it (#1623).

Reply:
Sounds good